tests/functional/states/test_freezer.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+pytestmark = [
+    pytest.mark.requires_salt_states("freezer.exampled"),
+]
+
+
+@pytest.fixture
+def freezer(states):
+    return states.freezer
+
+
+def test_replace_this_this_with_something_meaningful(freezer):
```

Comment:
**issue (blocking):** this file can be removed. it's an artifact from copier when you tell it to create certain module types. in our case, we've already pulled the tests over from the Salt core.